srcpkgs/volctl/template

```diff
@@ -1,13 +1,22 @@
 # Template file for 'volctl'
 pkgname=volctl
-version=0.8.0
-revision=2
+version=0.8.2
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-gobject desktop-file-utils"
-short_desc="Lightweight volume control for pulseaudio that sits in your systray"
+depends="python3-gobject desktop-file-utils python3-cairo python3-yaml python3-click"
+checkdepends="${depends} dbus pulseaudio gtk+3 xvfb-run"
+short_desc="Lightweight volume control for pulseaudio in your systray"
 maintainer="teldra <teldra@rotce.de>"
 license="GPL-2.0-only"
 homepage="https://buzz.github.io/volctl/"
 distfiles="https://github.com/buzz/volctl/archive/v${version}.tar.gz"
-checksum=01d021f244982a8bc827574f778a49864e937ca53310e03ec1386587bf8fb9b4
+checksum=8e744fe0e92f00451983e4fffa079d4d5354a8f96f6bcf8d15da44a18dc556f5
+
+post_extract() {
+	vsed -i "/^from gi.repository import Gtk.*/i import gi\ngi.require_version('Gtk', '3.0')" volctl/meta.py
+}
+
+do_check() {
+	xvfb-run dbus-run-session python setup.py test
+}
```

Comment:
Well, this at least guarantees we can import the whole thing, I guess.

If they ever add tests, we'll be ready to run them :D 

Maybe add a comment that currently no tests are defined? I definitely think `do_check` should remain like this. Possiby change `python` to explicit `python3` there too,